content/en/docs/demo/_index.md

```diff
@@ -1,5 +1,5 @@
 ---
-title: OpenTelemetry Demo Documentation
+title: OpenTelemetry Demo Docs
```

Comment:
Title tweak in passing; it felt too long otherwise! (I'd even consider dropping the "Docs", given that this is under the Docs section of the website :)).